FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/DataProcessor/ParquetDataProcessor.cs

```diff
@@ -76,7 +82,20 @@ public sealed class ParquetDataProcessor : IColumnDataProcessor
                 throw new ParquetDataProcessorException($"The FHIR schema node could not be found for schema type '{processParameters.SchemaType}'.");
             }
 
-            var inputStream = ConvertJsonDataToStream(processParameters.SchemaType, preprocessedData.Values);
+            // Convert data based on schema
+            JsonBatchData processedData;
+            if (IsCustomizedSchemaType(processParameters.SchemaType))
```

Comment:
Can we simplify this logic the check if it's not a default schema type"?

Reply:
Added ```resourceType``` in processParameters, and check if ```schemaType``` is equal to ```resourceType``` to recognize the default or custom.